client_end_user/shared/src/commonMain/kotlin/app/di/AppModule.kt

```diff
@@ -0,0 +1,60 @@
+package app.di
+
+import data.gateway.FakeGatewayImp
+import data.local.source.FakeLocalDataSourceImp
+import data.remote.service.FakeApiService
+import data.remote.service.FakeApiServiceImp
+import data.remote.source.FakeRemoteDataSourceImp
+import domain.gateway.FakeGateway
+import domain.source.FakeLocalDataSource
+import domain.source.FakeRemoteDataSource
+import domain.usecase.FakeUseCase
+import domain.usecase.FakeUseCaseImp
+import org.koin.core.Koin
+import org.koin.core.context.startKoin
+import org.koin.dsl.KoinAppDeclaration
+import org.koin.dsl.module
+import presentation.screens.HomeViewModel
+
+object AppModule {
+
+    fun initKoin(appDeclaration: KoinAppDeclaration = {}) = startKoin {
+        appDeclaration()
+        modules(
+            networkModule,
+            gatewayModule,
+            useCaseModule,
+            viewModelModule,
```

Comment:
It seems like the dataSourceModule is missing from the modules block.
```suggestion
        modules(
            networkModule,
            gatewayModule,
            useCaseModule,
            viewModelModule,
            dataSourceModule
```